dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -486,15 +503,20 @@ def runCrossValidation(self, jobId, interfaces):
             failures = Validator.crossValidateSql(comboRules.all(),submissionId)
             # get error file name
             reportFilename = self.getFileName(interfaces.errorDb.getCrossReportName(submissionId, row.first_file_name, row.second_file_name))
+            warningReportFilename = self.getFileName(interfaces.errorDb.getCrossWarningReportName(submissionId, row.first_file_name, row.second_file_name))
 
             # loop through failures to create the error report
-            with self.getWriter(regionName, bucketName, reportFilename,
-                                    self.crossFileReportHeaders) as writer:
+            with self.getWriter(regionName, bucketName, reportFilename, self.crossFileReportHeaders) as writer, \
+                 self.getWriter(regionName, bucketName, warningReportFilename, self.crossFileReportHeaders) as warningWriter:
                 for failure in failures:
-                    writer.write(failure[0:7])
+                    if failure[9] == interfaces.validationDb.getRuleSeverityId("fatal"):
```

Comment:
Been thinking about how we can reduce the number db calls. What do you think about doing a lookup the severity ids of "fatal" and "warning" set storing the as variables before the `for failure in failures:` loop starts? Or pulling back the values of the severity table as a dict? Either way, we'd then only have to do one set of database calls.